scripts/darwin.sh

```diff
@@ -49,20 +52,20 @@ for disk in $DISKS; do
     echo "mountpoints: []"
   else
     echo "mountpoints:"
-    echo "$mountpoints" | while read mountpoint ; do
+    echo "$mountpoints" | while read -r mountpoint ; do
       echo "  - path: $mountpoint"
     done
   fi
 
   echo "raw: $raw_device"
 
-  if [[ "$protected" == "Yes" ]]; then
+  if [ "$protected" == "Yes" ]; then
     echo "protected: True"
   else
     echo "protected: False"
   fi
 
-  if [[ "$device" == "/dev/disk0" ]] || \
+  if [ "$device" = "/dev/disk0" ] || \
      [[ ( "$removable" == "No" ) || ("$removable" == "Fixed") ]] || \
      [[ ( "$location" =~ "Internal" ) && ( "$removable" != "Yes" ) && ( "$removable" != "Removable" ) ]] || \
```

Comment:
shellcheck requires me to put `[[` in these cases because of `=~` and nested expressions (separated by parenthesis), but it requires me to switch to `[` for anything else.